### PR TITLE
Adds deployment config files for Hub Swagger UI

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
Signed-off-by: Puneet Punamiya <ppunamiy@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if functionality changed/added)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/main/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: swagger/config/00-swagger/00-swagger-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: swagger
+  namespace: tekton-hub
+data:
+  API_URL: 'https://api-tekton-hub.apps.ci-ln-1lv8f0k-d5d6b.origin-ci-int-aws.dev.rhcloud.com'
```
Comment (vinamra28) [inline swagger/config/00-swagger/00-swagger-configmap.yaml:8]: shouldn't this be `https://api.hub.tekton.dev` ?

File: swagger/config/01-kubernetes/01-swagger-ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+metadata:
+  name: swagger
+  namespace: tekton-hub
+  annotations:
+    acme.cert-manager.io/http01-edit-in-place: 'true'
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+spec:
+  tls:
+    - secretName: swagger-hub-tekton-dev-tls
+      hosts: [swagger-hub.tekton.dev]
```
Comment (vinamra28) [inline swagger/config/01-kubernetes/01-swagger-ingress.yaml:12]: shouldn't we keep this as `swagger.hub.tekton.dev`?

**Comment by sm43** [comment]
Maybe in the readme you can add the steps to deploy with files in pr.

**Review by vinamra28 — COMMENTED** [review]
/lgtm

**Pull request merged**